sqflite_common/lib/sqlite_api.dart

```diff
@@ -162,8 +188,11 @@ abstract class DatabaseExecutor {
   /// values from [whereArgs]
   ///
   /// Returns the number of rows affected if a whereClause is passed in, 0
-  /// otherwise. To remove all rows and get a count pass '1' as the
```

Comment:
I would remove, "pass '1' as the where clause" which is wrong. No where clause is sufficient